app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -13,9 +14,25 @@ const getStatusLink = text => {
   return allWords.filter(word => word.includes('https://t.co/'))[0];
 };
 
+const getMentionsAndTags = text => {
+  const allWords = text.split(/[\s,\n]+/);
+  return allWords.filter(word => word.includes('@') || word.includes('#'));
```

Comment:
We should have some test coverage on this. Could potentially consolidate the includes into the regex. Is this a valid mention `foor@bar`? Is this a valid tag `foo#bar`?